package.cmake

```diff
@@ -61,7 +61,7 @@ set(CPACK_DEBIAN_BASE_FILE_NAME "${CPACK_DEBIAN_FILE_NAME}.deb")
 string(REGEX REPLACE "^(${CMAKE_PROJECT_NAME})" "\\1-dev" CPACK_DEBIAN_DEV_FILE_NAME "${CPACK_DEBIAN_BASE_FILE_NAME}")
 
 #deb package tooling will be unable to detect deps for the dev package. llvm is tricky since we don't know what package could have been used; try to figure it out
-set(CPACK_DEBIAN_DEV_PACKAGE_DEPENDS "libboost-all-dev, libssl-dev, libgmp-dev, python3-numpy")
+set(CPACK_DEBIAN_DEV_PACKAGE_DEPENDS "libssl-dev, libgmp-dev, python3-distutils, python3-numpy, file, zlib1g-dev")
```

Comment:
you probably don't need `file` -- it's used for building the package, not for running anything in the package

Reply:
I think I do. Please look at [this](https://github.com/AntelopeIO/leap/actions/runs/5645037759/job/15291662020) failure.

Reply:
That's a defect in the libtester tests, should be fixed by #1444

Reply:
Are you going to remove `file` now that #1444 is merged?